TourMap/Onboarding/ViewModel/OnboardingViewModel.swift

```diff
@@ -0,0 +1,13 @@
+//
+//  OnboardingViewModel.swift
+//  TourMap
+//
+//  Created by Decagon on 19/01/2022.
+//
+
+import Foundation
+
+class OnboardingViewModel {
+  var data: [OnboardingData] = [OnboardingData(title: "Let's Travel", description: "\"Few places in this world are more dangerous than home. Fear not, therefore, to try the mountain passes. They will kill care, save you from deadly apathy, set you free, and call forth every faculty into vigorous, enthusiastic action.\" — John Muir", image: "travels1"), OnboardingData(title: "Plan A Trip", description: "\"It seems that the more places I see and experience, the bigger I realize the world to be. The more I become aware of, the more I realize how relatively little I know of it, how many places I have still to go, how much more there is to learn\" — Anthony Bourdain", image: "travels2"), OnboardingData(title: "Book A Flight", description: "\"Few places in this world are more dangerous than home. Fear not, therefore, to try the mountain passes. They will kill care, save you from deadly apathy, set you free, and call forth every faculty into vigorous, enthusiastic action.\" — John Muir", image: "travels3")]
```

Comment:
Move the string literals into a constant file. eg: instead of having "let's travel" here. You can move it to a constant file and using constant.title1 in this place. that way it will be easy for you to change thing later in one place. Avoid using string literals in code